utility/PlexPreferNonForcedSubs.py

```diff
@@ -0,0 +1,64 @@
+from plexapi.server import PlexServer
+import requests
+import xml.etree.ElementTree as ET
+#import pdb             //enable for debugging
+#pdb.set_trace()       //insert where to start debugging, press n to proceed to next line

```

Comment:
ElementTree is not used anywhere. Not sure the pdb library is necessary for debugging.